app/livechat/client/views/app/tabbar/chatRoomHistoryItem.html

```diff
@@ -0,0 +1,26 @@
+<template name='chatRoomHistoryItem'>
+	<li class='list-chat' id='{{_id}}' aria='{{v.token}}' >
+		<div class="history-div">
+			<div class="visitor-img">
+				<img src="/avatar/{{responseBy.username}}" alt="">
+			</div>
+			<div class="visitor-history-details">
+				<div class="visitor-details">
+					<h1>{{responseBy.username}}</h1>
+						{{#if open}}
+							<p class='open'>Open</p>
+						{{else}}
+							<p>Close at time {{time}}</p>
+						{{/if}}
+				</div>
+				<div class="total-messages">
+					<p>{{room.msgs}} messages</p>
```

Comment:
```suggestion
					<p>{{msgs}} messages</p>
```

Reply:
done but this also includes the one as a connected message and one as a promptscript so there will be always +2 then the original message